docs/pages/cldvideoplayer/configuration.mdx

```diff
@@ -42,6 +42,32 @@ import OgImage from '../../components/OgImage';
 | transformation     | object/array   | -               | Transformations to apply to the video    | `{ width: 200, height: 200, crop: 'fill' }` |
 | videoRef           | Ref            | -               | React ref to access video element        | See Refs Below               |
 | width              | string/number  | -               | **Required**: Player width               | `1920`                       |
+| aiHighlightsGraph  | boolean        | `false`         | **Check Cloudinary Video Player Api Docs**      | **Check Cloudinary Video Api Docs** |
```

Comment:
similarly here about the ABC order. again, if you think groupings make sense based on the cloudinary docs we can do that here

Reply:
Okay i will adress that right away, also i added a **"Note"** after my Description about being unable to run the docs locally due to the file path, do you have any idea how i might be able to resolve that so i can preview the docs to confirm its displaying properly